src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -1081,6 +1084,11 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 reportSuppressedDiagnostics: reportSuppressedDiagnostics
             );
 
+            if (debugPlus)
+            {
+                options = options.WithDebugPlusMode(debugPlus);
```

Comment:
VB is NYI. 
I'd like to make sure this change is sufficient for C# before doing the same in VB. In this chance I tried not to touch VB except where necessary because of shared common code.